api/v1/routes/auth_facebook.py

```diff
@@ -0,0 +1,82 @@
+import uuid
+
+from fastapi import APIRouter, Depends, status
+
+from sqlalchemy.orm import Session
+from api.db.database import get_db
+
+from decouple import config
+from datetime import timedelta
+from typing import Annotated
+
+from api.core.responses import INVALID_CREDENTIALS, COULD_NOT_VALIDATE_CRED
+
+from api.utils.auth import create_access_token
+from api.utils.json_response import JsonResponseDict
+
+from api.v1.services.facebook import validate_facebook_token, create_db_user, create_oauth_user
+
+from api.v1.schemas.facebook import userToken
+
+# don't remove the following imports
+from api.v1.models.user import User
+from api.v1.models.oauth import OAuthUser
+from api.v1.models.profile import Profile
+from api.v1.models.org import Organization
+# -------------------------------
```

Comment:
everything is in the `models/__init__.py` file, import from there `from models import User, Profile ....`

Reply:
I'm working on it